### ISSUE TITLE
MacOS Support 

### ISSUE THREAD

**Issue by Elkay14** [jasarien/Provenance#661]
Is their a way to Run Provenance on your Mac?

**Comment by XeresRazor** [comment]
Yes, it’s called OpenEmu. www.openemu.org

**Comment by OlliM** [comment]
@jasarien - Would it make sense to re-open this to add support to Apple Silicon/M1? Openemu runs in Rosetta 2...

**Comment by jasarien** [comment]
I don’t believe that OpenEmu will be in Rosetta 2 for long. Apple aren’t keeping it around for very long (just like the first Rosetta). 

The effort required to turn Provenance into a functional and user-friendly macOS app would be a lot.
